src/main/java/frc/robot/Robot.java

```diff
@@ -119,8 +119,7 @@
   public void robotInit() {
     // Zero all nesscary sensors on Robot
     Config.getInstance().reload();
-
```

Comment:
Did you mean to remove the logging code?

Reply:
Yes